freqtrade/configuration/arguments.py

```diff
@@ -55,8 +55,14 @@
 ARGS_PLOT_PROFIT = ["pairs", "timerange", "export", "exportfilename", "db_url",
                     "trade_source", "ticker_interval"]
 
+ARGS_HYPEROPT_LIST = ["hyperopt_list_best", "hyperopt_list_profitable", "print_colorized",
+                      "print_json", "hyperopt_list_no_details"]
+
+ARGS_HYPEROPT_SHOW = ["hyperopt_list_best", "hyperopt_list_profitable", "hyperopt_show_index",
+                      "print_json", "hyperopt_show_no_header"]
+
 NO_CONF_REQURIED = ["download-data", "list-timeframes", "list-markets", "list-pairs",
-                    "plot-dataframe", "plot-profit"]
+                    "hyperopt_list", "hyperopt_show", "plot-dataframe", "plot-profit"]
```

Comment:
these should be the command names ... 

Fixing this should also get rid of the test error.

However - to test without a config, i would also use the following snippet - to avoid using a config if it exists, causing random test-failures. We use this in other tests that don't require a config (download-datadir, ...) within `test_utils.py`.

```python
    pargs = get_args(args)
    pargs['config'] = None
```

Reply:
Thanks, fixed.